<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCardTraveler</class>
 <widget class="QWidget" name="FormCardTraveler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLable{
font: 10pt &quot;Arial&quot;;
color: rgb(180, 80, 180);
background-color:rgb(14, 130, 255);
border-style: outset;
border-width: 0px;
}</string>
  </property>
  <widget class="QLabel" name="yourPhoto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(35, 123, 255);
font: 10pt &quot;Arial&quot;;
border-style: outset;
border-width: 0px;</string>
   </property>
   <property name="text">
    <string>Ваше Фото</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addPhotoBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgSlot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(35, 123, 255);</string>
   </property>
   <property name="text">
    <string>Копия паспорта</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imgSlot_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(35, 123, 255);</string>
   </property>
   <property name="text">
    <string>Электронный билет</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addImgBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addImgBtn_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ВОЗРАСТ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>МЕСТО ЖИТЕЛЬСТВА</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>380</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Card Travel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дополнительные Заметки</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
